docs/user/guide-intermediate.md

```diff
@@ -0,0 +1,331 @@
+# Guide to intermediate-level Cursorless
+
+Starting to feel comfortable with the basics? Great 🙌! Let's get you to the next level. This document will do the following:
+
+- Highlight the actions and modifiers in Cursorless that the pros use most.
+- Teach you a few important concepts that underlie a lot of Cursorless behavior.
+- Walk through some example editing operations that are common but perhaps tricky to a beginner.
+
+This document assumes that you understand the very basics of Cursorless, like `"take air"` and `"chuck line"`.
+If those commands don't mean anything to you, we'd advise having a look at of [the main documentation page](README.md) and watching [the Cursorless tutorial videos](https://www.youtube.com/watch?v=5mAzHGM2M0k&list=PLXv2sppxeoQZz49evjy4T0QJRIgc_JPqs) to get off the ground.
+
+We plan to add basic and advanced levels before and after this guide; for now the [the main documentation page](README.md) and the YouTube videos are the best place to learn basic / advanced Cursorless.
+
+Note that we use the following conventions here:
+
+- Words in square braces (`[]`) are optional, eg `hello [world]` means you can say `"hello"` or `"hello world"`.
+- Slashes (`/`) show alternatives. For example, `hello/world` means you can say `"hello"` or `"world"`.
+- Targets are represented by `T` with a possible digit, eg `T` or `T1` or `T2`. For example, `bring T1 to T2` means you can say `"bring air to bat"`.
```

Comment:
Do we want to include examples in this section? I like the idea of being example-heavy

Reply:
Examples of what?

Reply:
If you mean examples of the `[T/T1/T2]` syntax, what you have seems good.